### PR TITLE
KYLIN-3597 Improve code smell

### PR DESCRIPTION
**Pull request by hit-lacus**
1. refactor logger format
2. use try with resource other than closeQuietly method of IOUtils

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

File: core-storage/src/main/java/org/apache/kylin/storage/gtrecord/SegmentCubeTupleIterator.java [hunk 1]
```diff
@@ -98,14 +98,23 @@ public GTInfo getInfo() {
...
                 }
             };
-            GTStreamAggregateScanner aggregator = new GTStreamAggregateScanner(inputScanner, scanRequest);
-            return aggregator.valuesIterator(gtDimsIdx, gtMetricsIdx);
+            Iterator<Object[]> result;
+            try (GTStreamAggregateScanner aggregator = new GTStreamAggregateScanner(inputScanner, scanRequest)) {
+                result = aggregator.valuesIterator(gtDimsIdx, gtMetricsIdx);
+            } catch (IOException ioe) {
+                // implementation of close method of anonymous IGTScanner is empty, no way throw exception
+                throw new RuntimeException("IOException is not expected here.", ioe);
```
Comment (shaofengshi) [inline core-storage/src/main/java/org/apache/kylin/storage/gtrecord/SegmentCubeTupleIterator.java]: RuntimeException should be avoided; use a specific exception.

File: source-kafka/src/main/java/org/apache/kylin/source/kafka/config/KafkaConsumerProperties.java [hunk 2]
```diff
@@ -101,24 +100,22 @@ public static Properties extractKafkaConfigToProperties(Configuration configurat
...
-                properties.putAll(extractKafkaConfigToProperties(oconf));
-                IOUtils.closeQuietly(ois);
+                try (FileInputStream ois = new FileInputStream(propOverrideFile)) {
+                    Configuration oconf = new Configuration();
+                    oconf.addResource(ois);
+                    properties.putAll(extractKafkaConfigToProperties(oconf));
+                }
             }
         } catch (IOException e) {
             throw new RuntimeException(e);
```
Comment (shaofengshi) [inline source-kafka/src/main/java/org/apache/kylin/source/kafka/config/KafkaConsumerProperties.java]: The same as previous comment.

**Pull request merged**